ci/requirements-conversion.txt

```diff
@@ -2,8 +2,10 @@ absl-py==0.8.0            # via tensorboard, tensorflow
 astor==0.8.0              # via tensorflow
 certifi==2019.9.11        # via requests
 chardet==3.0.4            # via requests
+click==7.0
```

Comment:
`click` shouldn't be required anymore (in fact, I see you've already removed it from the `.in` file).